### DISCUSSION TITLE
French & Portuguese models spams non-verbal subs instead of creating them.

### DISCUSSION THREAD

**Discussion by StickyChannel92** [SubtitleEdit/subtitleedit#8784] [Q&A]
I'm trying to create subs for a TV series I archived, but unfortunately, Subtitle Edit does not play nice with French and Portuguese languages in the Whisper Const-me model. All it did was spam _[music]_ all the way down to the end or at a certain point. The CPP and CPP cuBLAS take forever and Vosk is not that accurate and does not translate.

If anyone can help me figure out what's wrong, I'd appreciate it, as I'd love to watch the series I archived! :]

Video used: https://mega.nz/file/17kjFSLL#_qBp45dVZXt_d3rpwnQSyOhN-mI1wuvp2f8gnmtKPMI

Windows 10 Pro
Intel Xeon E5-1620 v3 3.5GHz
32GB RAM
NVIDIA Geforce RTX 2060 SUPER
Subtitle Edit version 4.0.7
Model used: Const-me large.bin

**Comment by niksedk** [comment]
Try with `Purfview Faster Whisper` with `large-v2` model - is that better?

**Reply by StickyChannel92** [reply]
Initially:
```
That seems to work much better. There are still errors in the script, but it works like a charm. Just don't forget to close Opera GX and Aseprite (and all resource-hungry apps); otherwise, the computer would black out with a red power button (5 beeps).

I originally had to install CUDA and CUDNN kits but unfortunately, I was wasting nearly 10GB of data and hours of trying to open the CUDA installer just to install them, and didn't solve the dll problem directly. Luckily, I had a CUDNN package in SVP4 and it works like a charm! Just stick it into the same directory where whisper-faster.exe is and you're good to go!

I even tried ctranslate2 (with both the large-v2 and medium models, and installed with python), and it had a few more errors in the script, but still works better than Const-me. Again, the CUDNN package goes in "C:\Users\<USERNAME>\AppData\Local\Programs\Python\<PYTHONVERSION>\Lib\site-packages\ctranslate2\".
```

EDIT 8/31/2024 9:09 PM CST: My computer sparked while doing a batch script and that caused the computer to black out and do 5 beeps (HP machine). I was able to start up the computer again with 10 subs done. What gives?
